epicmail/app/views/message_views.py

```diff
@@ -3,8 +3,9 @@
 from flask import Blueprint, request, make_response, jsonify
 from flask.views import MethodView
 from epicmail.app.views.user_views import LoginUser
-from epicmail.app.models.messages import Messages, messages
-# from epicmail.app.handler.auth import JwtAuth
+from epicmail.app.models.messages import Message
+from epicmail.app.handler.validators.message_validators import (
+    validate_address, validate_createdby, validate_message, validate_subject)
 
 # authentic = JwtAuth()
```

Comment:
remove the comments if there don't add any value to our code, please.

Reply:
check